modules/sharedaddy/sharing.js

```diff
@@ -20,12 +20,8 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 				}
 
 				// get both the http and https version of these URLs
-				https_url = encodeURIComponent( url.replace( /^http:\/\//i, 'https://' ) );
```

Comment:
In 7b2e79d, you had consolidated the `encodeURIComponent` from individual requests to encode as part of the `https_url` and `http_url` assignments. You'll either want to push this back to WPCOM (and add back to [lines 23/24](https://github.com/Automattic/jetpack/pull/2119/files#diff-1ef3c76357b55ce0f40b6c21eab8b4bfR23)), or reinstate the `encodeURIComponent` calls when generating URLs for the `urls` object.